### PR TITLE
Create Random_Forest.py

### PR DESCRIPTION
**Pull request by jahnvigupta**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by shreyasbapat** [comment]
As the PR seems quite unattended, I will take it for here. 

@jahnvigupta ,would you mind describing what exactly are you trying to do here? I am not in touch so!

**Comment by jahnvigupta** [comment]
I have trained the model using random forest and calculated the accuracy on test data which came out to be .99 .

**Comment by shreyasbapat** [comment]
Alright I am a bit skeptical about the results. Do you need me to merge the Pull Request?

File: Random_Forest.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                        Labels.append(bool(int(line.split(',')[0]) - 1))
+        return np.array(Labels)	
+
+data = data.dropna(axis='columns', thresh = int(0.5 * len(data)))
+data.fillna(data.mean())
+col = data.columns
+X = data[col[1:]]
+y = data['LABEL']
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3)
+from sklearn.ensemble import RandomForestClassifier
```
Comment (shreyasbapat) [inline Random_Forest.py]: Alright, general practice, put all the imports up above the code.

File: Random_Forest.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+data = pd.read_csv("train") #train is location of training data
```
Comment (shreyasbapat) [inline Random_Forest.py]: Put this down somewhere.

File: Random_Forest.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+data = data.dropna(axis='columns', thresh = int(0.5 * len(data)))
+data.fillna(data.mean())
+col = data.columns
+X = data[col[1:]]
+y = data['LABEL']
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3)
+from sklearn.ensemble import RandomForestClassifier
+
+# Create the model with 100 trees
+model = model = RandomForestClassifier(n_estimators=100,
```
Comment (shreyasbapat) [inline Random_Forest.py:16]: Typo here!

**Review by shreyasbapat — CHANGES_REQUESTED** [review]
I suggest you to install :

`pip install isort black`

And then run:

`black Random_Forest.py`

and 

`isort Random_Forest.py` 

And then, the code would be much more readable!

**Comment by shreyasbapat** [comment]
Oh, and one more piece of advice, never and I repeat **never** use `master` branch to create merge requests! 
Always create a new branch every time you want to make new changes.

**Comment by ghost** [comment]
@jahnvigupta are  you still working on this?

**Comment by abhijeetmanhas** [comment]
@jahnvigupta can you update the PR according to recent template?

**Comment by LalitNM** [comment]
I am merging this as this doesn't have a conflict. I am going to rearrange some files in repository ([this](https://github.com/STAC-IITMandi/Exoplanet-Hunt/blob/master/Bishop%20-%20Pattern%20Recognition%20And%20Machine%20Learning%20-%20Springer%20%202006.pdf) and [this](https://github.com/STAC-IITMandi/Exoplanet-Hunt/blob/master/duda-etal-00.pdf) should be in a separate directory, maybe in `/docs`).
